### DISCUSSION TITLE
太好用了，有pr到官方的计划吗？

### DISCUSSION THREAD

**Discussion by atticus-lv** [CrystalWindSnake/ex4nicegui#181] [Q&A]
就单一个to_ref就很爽了，给我一种用kotlin compose的感觉，思考起来很流畅，用起来也很方便

**Comment by CrystalWindSnake** [comment]
构建响应式系统需要太多组件内部的支持，感觉不太适合合并到nicegui上。目前来看，单独维护一个库，还是可以的。😁
